packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -121,6 +121,17 @@ const COLUMNS_VM_PV = [
       return sr !== undefined && sr.name_label
     },
   },
+  {
+    itemRenderer: (vdi, userData) => {
+      const vbd = userData.vbdsByVdi[vdi.id]
```

Comment:
Can it be `undefined`?

Reply:
No, because this is the `VBD` for the `VM-VDI` pair.

Reply:
Then inline it ;)